r/src/cluster.r

```diff
@@ -17,7 +17,8 @@ getClusters <- function(req){
     #In the meta data slot the clustering is stored with the resolution used to calculate it
     # RNA_snn_res.#resolution
     #
-    str <- paste("RNA_snn_res.",toString(resol),sep = "")
+
+    str <- paste(data@active.assay,"_snn_res.",toString(resol),sep = "")
```

Comment:
This change was made based on recommendation from bioinformaticians for it to work adequately according to the experiment picked.

Reply:
do you know what does it do? I am just curious

Reply:
It changes the string used depending whether we are working with a multiple samples experiment or a single sample one.

I don't know much else about it, but @ogibson-biomage might be able to provide more context.

Reply:
that sounds good enough to me, thanks!

Reply:
We need to acces the column in the dataframe that is named RNA_snn_res or integrated_snn_res, depending on the type of experiment, so we access the active assay slot of the object to figure out if it's a multisample integrated experiment and modify the string for searching the column accordingly!